jib-core/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ All notable changes to this project will be documented in this file.
 ### Changed
 
 - `TarImage` is constructed using `TarImage.at(...).named(...)` instead of `TarImage.named(...).saveTo(...)` ([#1918](https://github.com/GoogleContainerTools/jib/issues/1918))
+- For retrieving credentials from Docker config (`~/.docker/config.json`), `credHelpers` now takes precedence over `credsStore`, followed by `auths`. Moreover, the legacy `credsStore` no longer requires defining empty registry entries in `auths` to be used. This means that if `credsStore` is defined, `auths` will be completely ignored. ([#1958](https://github.com/GoogleContainerTools/jib/pull/1958))
```

Comment:
I feel like our changelog entries are getting longer and longer, haha. Maybe we can split this into two (`credStore` change different from `credHelpers` precedence)